packages/block-editor/src/hooks/index.native.js

```diff
@@ -5,3 +5,4 @@ import './custom-class-name';
 import './generated-class-name';
 import './style';
 import './color';
+import './font-size';
```

Comment:
Sérgio, do we need this addition since the code that uses it is guarded for web only?

Reply:
We do because that will add hooks that are used to save and load the props properly. If we don't add it the attributes related to font-size that come from web posts, will be lost/erased by gb-mobile.